src/Console/Migrate.php

```diff
@@ -123,8 +199,31 @@ protected function resolveBuilders(): void
         $this->target = $this->resolveBuilder($this->target());
     }
 
+    protected function resolveOptions(): void
+    {
+        $this->tableNames =   $this->tableNames();
+        $this->excludeTables =   $this->excludeTables();
+
+        if ($this->tableNames[0] == '-' && $this->confirm('Get all table list from target connection(incase if source connection does not support it)?', false))
```

Comment:
1. Use strict comparison - `===`.
2. Move the question to a separate method.
3. Missing a space character in the question.
4. The logical block is not wrapped in curly braces.